homeassistant/components/climate/venstar.py

```diff
@@ -0,0 +1,281 @@
+
+    add_devices([VenstarThermostat(client)], True)
+
+
+class VenstarThermostat(ClimateDevice):
+    """Representation of a Venstar thermostat."""
+
+    def __init__(self, client):
+        """Initialize the thermostat."""
+        self._client = client
+        self._fan_list = VALID_FAN_STATES
+        self._operation_list = VALID_THERMOSTAT_MODES
+
+    def update(self):
+        """Update the data from the thermostat."""
+        _LOGGER.info("Refreshing data from your Venstar Thermostat.")
+        info_success = self._client.update_info()
+        sensor_success = self._client.update_sensors()
+        if not info_success or not sensor_success:
+            _LOGGER.error("Failed to update data from your Thermostat.")
+
+# Thermostat config
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        features = (SUPPORT_TARGET_TEMPERATURE |
+                    SUPPORT_FAN_MODE |
+                    SUPPORT_OPERATION_MODE)
+
+        if self._client.mode == self._client.MODE_AUTO:
+            features |= (SUPPORT_TARGET_TEMPERATURE_HIGH |
+                         SUPPORT_TARGET_TEMPERATURE_LOW)
+
+        if self._client.hum_active == 1:
+            features |= SUPPORT_TARGET_HUMIDITY
+
+        return features
+
+    @property
+    def name(self):
+        """Return the name of the thermostat."""
+        return self._client.name
+
+    @property
+    def precision(self):
+        """Return the precision of the system.
+
+        Venstar temperature values are passed back and forth in the
+        API as whole degrees C or F.
+        """
+        return PRECISION_WHOLE
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement, as defined by the API."""
+        if self._client.tempunits == self._client.TEMPUNITS_F:
+            return TEMP_FAHRENHEIT
+        else:
+            return TEMP_CELSIUS
+
+    @property
+    def fan_list(self):
+        """Return the list of available fan modes."""
+        return self._fan_list
+
+    @property
+    def operation_list(self):
+        """Return the list of available operation modes."""
+        return self._operation_list
+
+# Current Values
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        return self._client.get_indoor_temp()
+
+    @property
+    def current_humidity(self):
+        """Return the current humidity."""
+        return self._client.get_indoor_humidity()
+
+    @property
+    def current_operation(self):
+        """Return current operation ie. heat, cool, idle."""
+        if self._client.mode == self._client.MODE_HEAT:
+            return STATE_HEAT
+        elif self._client.mode == self._client.MODE_COOL:
+            return STATE_COOL
+        elif self._client.mode == self._client.MODE_AUTO:
+            return STATE_AUTO
+        else:
+            return STATE_IDLE
```

Comment:
It's a little confusing because the thermostat is still "on".  It's screen is on, it's still monitoring/reporting temperature/humidity, still responding to API calls, etc.  

But your assumption is correct there- it will not turn on the air conditioner, nor the furnace to attempt to keep a certain temperature.

It looks like in the honeywell component they're referring to this state as "idle" (see line 286 from honeywel.py).  Looks like nuheat is also.

Reply:
Okay… we should probably clean that up at some point, but for now this is probably fine. :)

Reply:
Oh- no problem if you'd rather use the "off" state to represent this.  I'll go ahead and do that.